config.toml

```
@@ -36,6 +36,12 @@ ignoreFiles = [
 # Hugo - Top-level navigation (horizontal)
 ###############################################################################
 [menu]
+  [[menu.main]]
+    name = "Kubeflow Summit 2023 "
+    weight = -1000
+    pre = "<i class='fas fa-calendar pr-2' style='color: #FFC107'></i>"
+    post = "<br><span class='badge badge-warning'>Oct 6th</span> <span class='badge badge-warning'>Irving, Texas</span> <span class='badge badge-warning'>Virtual</span>"
```

Should we say Virtual/In-Person at the top so people can see it clearly ?
```suggestion
    post = "<br><span class='badge badge-warning'>Oct 6th</span> <span class='badge badge-warning'>Irving, Texas</span> <span class='badge badge-warning'>Virtual/In-Person</span>"
```